### ISSUE TITLE
Suggestion: Conversion to integer

### ISSUE THREAD

**Issue by scheinerman** [anthonyclays/RomanNumerals.jl#12]
Fun package. Thank you. One easy suggestion: Add a function that will covert a `RomanNumeral` to an integer. This works:
```
Int(x::RomanNumeral) = x.val
```

**Comment by hhaensel** [comment]
You can already do `convert(Int, rn"xiv")`, furthermore you can do arithmetics with RomanNumerals. If you have one regular number involved, the output will be of that type. That makes it possible to do the following:
```julia
julia> 1rn"x"
10

julia> 1.0rn"x"
10.0
```
For the next version, I will include
```julia
(::Type{T})(x::RomanNumeral) where T<:Number = convert(T, x)
```

**Comment by hhaensel** [comment]
included in version v0.3.3
